docs/content/timeline/changelog.md

```diff
@@ -15,16 +15,25 @@ weight = 11
 
 ## Recent Updates
 
-* Added ability to have regex-based SPIFFE ID matchers.
+* Added the ability to have regex-based SPIFFE ID matchers.
 * Enabled stricter validation on SPIFFE IDs to reduce configuration errors.
-* Added ability to optionailly use multiple worker nodes for the development
+* Added ability to optionally use multiple worker nodes for the development
   clusters.
+* Introduced `helm-docs` to automatically augment the documentation with the
+  Helm chart's values.yaml.
+* Added the ability to deploy VSecM without SPIRE Controller Manager. In this
```

Comment:
The “VSecM w/o SCM” and “VSecM w/o clusterspiffeids” use cases still need integration testing.

In theory, it should work; and in theory it should make some of the users’ (read: Tomasso) life much better :) .